ci/rh-olm-release-pipeline.yaml

```diff
@@ -0,0 +1,400 @@
+
+jobs:
+  - name: self-update
+    max_in_flight: 1
+    plan:
+      - get: agent-operator-pipeline-source
+        trigger: true
+      - set_pipeline: self
+        file: agent-operator-pipeline-source/ci/rh-olm-release-pipeline.yaml
+
+  - name: olm-release-pr-creation
+    max_in_flight: 1
+    plan:
+      - get: instana-agent-operator-release
+        params:
+          globs:
+            - olm-*.zip
+        trigger: true
+      - in_parallel:
+          steps:
+            - do:
+                - get: community-operators-repo
+                - task: community-operator-commit-changes
+                  config:
+                    platform: linux
+                    image_resource:
+                      type: registry-image
+                      source:
+                        repository: registry.access.redhat.com/ubi8/ubi-minimal
+                    inputs:
+                      - name: instana-agent-operator-release
+                      - name: community-operators-repo
+                    outputs:
+                      - name: community-operators-repo
+                    params:
+                      GH_API_TOKEN: ((instanacd-github-api-token))
+                      OWNER: k8s-operatorhub
+                      REPO: community-operators
+                    run:
+                      path: bash
+                      args:
+                        - -ce
+                        - |
+                          microdnf install git zip curl jq
+                          
+                          pushd instana-agent-operator-release
+                          
+                          OLM_BUNDLE_ZIP=$(ls olm*.zip)
+                          OPERATOR_RELEASE_VERSION=$(echo $OLM_BUNDLE_ZIP | sed 's/olm-\(.*\)\.zip/\1/')
+                          COMMIT_MESSAGE="operator instana-agent-operator ($OPERATOR_RELEASE_VERSION)"
+                          OLM_BUNDLE_ZIP_PATH="$(pwd)/$OLM_BUNDLE_ZIP"
+                          
+                          # If the PR is already open then we are done
+                          PR_LIST_JSON=$(curl -fH "Accept: application/vnd.github+json" https://api.github.com/repos/$OWNER/$REPO/pulls)
+                          EXISTING_PR_INFO_JSON=$(echo $PR_LIST_JSON | jq ".[] | select(.title==\"$COMMIT_MESSAGE\")")
+                          if [ ! -z "$EXISTING_PR_INFO_JSON" ]; then
+                            echo "PR already exists, exiting"
+                            exit 0
+                          fi
+                          
+                          popd
+                          
+                          pushd community-operators-repo/operators/instana-agent-operator
+                          
+                          mkdir -p $OPERATOR_RELEASE_VERSION
+                          unzip -o $OLM_BUNDLE_ZIP_PATH -d $OPERATOR_RELEASE_VERSION
+                          
+                          git config --global user.name "instanacd"
```

Comment:
we should not hardcoded username and email. Please use the Secret Manager. Also it is public github repository, so we have to hide these informations.

Reply:
Concourse requires us to bind secrets like this: `((instanacd-github-api-token))`, so the user is already hardcoded and visible in a sense, isn't it? I can make `((instanacd-github-username))` and `((instanacd-github-email))`, but that information will be visible anyway by looking at the instancd github account which will be attached to the PRs that get opened...

Reply:
You are right about both. One benefit for using Secret Manager would be rotation of both entries.
And with the naming, we could use a more generic name without the instanacd. I will ask tomorrow other teams about the naming, maybe it is okay to hardcode both